backend/api/v1/endpoints/research.py

```
@@ -0,0 +1,373 @@
+            author=author,
+            research_field=research_field,
+            file_path=file_location,
+            uploader_id=current_user.id  # ✅ Link paper to uploader
+        )
+        db.add(new_paper)
+        db.commit()
+        db.refresh(new_paper)
+
+        return {"message": "Paper uploaded successfully", "paper_id": new_paper.id, "file_path": file_location}
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)